package.json

```diff
@@ -87,6 +87,7 @@
     "firefox-client": "0.3.0",
     "flow-bin": "0.33.0",
     "git-hooks": "1.1.1",
+    "git-hooks-win": "1.1.11",
```

Comment:
Good point, I didn't want to mess with the original package since I don't have a mac to try if this one works properly on Mac too. If it does we should remove the other one and keep the windows compatible  ,otherwise I'll try one of the few other solutions I looked into (like grunt-git-hooks) which are a bit more complicated.